demo_policy/e2e.bats

```diff
@@ -1,5 +1,58 @@
 #!/usr/bin/env bats
 
+@test "should allow pod creation in default namespace" {
+    run kwctl run annotated-policy.wasm -r ./test_data/no_privileged_containers.json --settings-json '{"testScenario": "can-i-success"}' --replay-host-capabilities-interactions ./test_data/sessions/can-i-success.yml --allow-context-aware
+    echo "output = ${output}"
+    [ "$status" -eq 0 ]
+    [ $(expr "$output" : '.*allowed.*true') -ne 0 ]
+    [[ "$output" =~ '"allowed":true' ]]
+}
+
+@test "should deny node deletion" {
```

Comment:
This is really confusing as name, I doubt the request is really about deleting a `Node` resource given its name.